samples/client/petstore/swift5/validation/PetstoreClient/Classes/OpenAPIs/APIs.swift

```diff
@@ -5,14 +5,6 @@
 //
 
 import Foundation
-
-// We reverted the change of PetstoreClientAPI to PetstoreClient introduced in https://github.com/OpenAPITools/openapi-generator/pull/9624
-// Because it was causing the following issue https://github.com/OpenAPITools/openapi-generator/issues/9953
-// If you are affected by this issue, please consider removing the following two lines,
-// By setting the option removeMigrationProjectNameClass to true in the generator
-@available(*, deprecated, renamed: "PetstoreClientAPI")
-public typealias PetstoreClient = PetstoreClientAPI
-
```

Comment:
I don't know why these changes are only in my branch, they appeared when I generated samples